.travis.yml

```diff
@@ -7,7 +7,12 @@ install:
    - travis_retry composer install --no-interaction --no-suggest
 
 script:
-  - composer test -- --coverage-clover=coverage.xml
+  - composer test -- --log-junit=phpunit.junit.xml --coverage-clover=coverage.xml
+  - sonar-scanner
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+
```

Comment:
Add somewhere
```
cache:
  directories:
- '$HOME/.sonar/cache'
```

Reply:
what's the point of those lines ?

Reply:
I'll skip those atm and add them later if needed